roles/remove_ztp_gitops_resources/tasks/main.yaml

```diff
@@ -0,0 +1,98 @@
+---
+
+# This will prevent ArgoCD from recreating the spoke cluster
+# resources if we decide to detach the spoke cluster from a
+# given hub.
+- name: Delete SiteConfig and Policy Applications
+  community.kubernetes.k8s:
+    api_version: argoproj.io/v1alpha1
+    kind: Application
+    name: "{{ _rzgr_gitops_application }}"
+    namespace: openshift-gitops
+    state: absent
+    wait: true
+    wait_sleep: 5
+    wait_timeout: 120
```

Comment:
nit: These are the default `wait_sleep` and `wait_timeout` values, no need to set them unless they are different or will be modified by a variable

Reply:
ack